web/src/app/first-login/page.tsx

```diff
@@ -0,0 +1,25 @@
+import CompanyLogo from "../components/company-logo";
+import FirstLoginForm from "./components/first-login-reset-password";
+
+export default async function FirstLogin() {
+    return (
+        <section className="py-12 px-6 space-y-12">
+            <div className="flex flex-col items-center">
+                <p className="font-semibold">Bem vindo ao</p>
+                <CompanyLogo />
+            </div>
+
+            <div className="items-center">
+                <img src="/images/welcome.svg" alt="Imagem de boas vindas" />
```

Comment:
Aqui tem duas coisas.

1. Importe a imagem como componente, assim a gente vai poder ter esse asset bem próximo de onde ele é usado. Vide o último PR da Jaqueline pra ver como foi feito.
2. Alterar o alt pra descrever o que tem na imagem
```suggestion
                <img src="/images/welcome.svg" alt="Desenho de 3 pessoas celebrando com uma faixa de boas-vindas pendurar acima deles" />
```

Reply:
Como eu faço para adicionar o alt importando a imagem como componente?